Compute/tests/Unit/V1/AcceleratorTypesClientTest.php

```diff
@@ -101,8 +101,7 @@ public function aggregatedListTest()
         $resources = iterator_to_array($response->iterateAllElements());
         $this->assertSame(1, count($resources));
         $this->assertArrayHasKey('itemsKey', $expectedResponse->getItems());
-        $this->assertArrayHasKey('itemsKey', $resources);
-        $this->assertEquals($expectedResponse->getItems()['itemsKey'], $resources['itemsKey']);
+        $this->assertEquals($expectedResponse->getItems()['itemsKey'], $resources[0]);
```

Comment:
@miraleung  This change is causing the tests to fail. Why did this get reverted back? 

Aggregated lists return maps, so this should be using the `'itemKey'` string instead of the index zero.